kms/yubikey/yubikey.go

```diff
@@ -190,6 +218,35 @@ func (k *YubiKey) CreateSigner(req *apiv1.CreateSignerRequest) (crypto.Signer, e
 	return signer, nil
 }
 
+// CreateAttestation creates an attestation certificate from a YubiKey slot.
+//
+// # Experimental
+//
+// Notice: This API is EXPERIMENTAL and may be changed or removed in a later
+// release.
+func (k *YubiKey) CreateAttestation(req *apiv1.CreateAttestationRequest) (*apiv1.CreateAttestationResponse, error) {
+	slot, err := getSlot(req.Name)
+	if err != nil {
+		return nil, err
+	}
+
+	cert, err := k.yk.Attest(slot)
```

Comment:
Callers of this function would call this after `k.yk.GenerateKey` with the name of the slot in which the key was created, correct?

Reply:
For example, you can create a key using:

```console
$ step kms create yubikey:slot-id=85
-----BEGIN PUBLIC KEY-----
...
-----END PUBLIC KEY-----
$ step kms attest yubikey:slot-id=85
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
-----BEGIN CERTIFICATE-----
...
-----END CERTIFICATE-----
```